src/Server/RestRouterHandlers/TableRestRouterHandler.cpp

```diff
@@ -78,11 +79,29 @@ bool TableRestRouterHandler::validatePost(const Poco::JSON::Object::Ptr & payloa
         }
     }
 
+    int replication_factor = payload->has("replication_factor") ? payload->get("replication_factor").convert<Int32>() : 1;
```

Comment:
This is not a correct way to check the broker size

Reply:
Can we defer this enhancement ?

Reply:
OK, let‘s defer this error handle. And I will rebase the code and add the fix after PR #145 merged.